pages/tools/index.js

```diff
@@ -0,0 +1,37 @@
+import ToolDashboard from '../../components/tools/ToolDashboard';
+import ToolFilter from '../../context/ToolFilterContext';
+import GenericLayout from '../../components/layout/GenericLayout';
+import Heading from '../../components/typography/Heading';
+import Paragraph from '../../components/typography/Paragraph';
+import Container from '../../components/layout/Container';
+import TextLink from '../../components/typography/TextLink';
+
+export default function toolsDashboard() {
+  const description = 'Tools Dashboard for AsyncAPI Initiative';
+  const image = '/img/social/meetings.png';
```

Comment:
We need new image for new page, please remember about that.

Reply:
Means? Should I copy this image file and rename it as `tools.png`?

Reply:
But almost every page has the same AsyncAPI logo shown in the tab as the favicon, then why create a new one?